caffe2/operators/elementwise_linear_op.cc

```diff
@@ -158,7 +158,8 @@ print("Y:\n", workspace.FetchBlob("Y"))
     .Input(1, "w", "1D scaling factors, or weights, of size $D$. This input contains the weights that will be multiplied by the data.")
     .Input(2, "b", "1D biases of size $D$. This input contains the biases that will be added to the products of the weights and data.")
     .Output(0, "Y", "2D output tensor of size $NxD$. Calculated as described above.")
-    .Arg("axis", "*(type: int; default: 1)* Describes the axis of the inputs; defaults to one because the 0th axis most likely describes the batch size.");
+    .Arg("axis", "*(type: int; default: 1)* Describes the axis of the inputs; defaults to one because the 0th axis most likely describes the batch size.")
+    .InheritOnnxSchema();
```

Comment:
I think you have to give the corresponding ONNX operator name in InheritOnnxSchema

Reply:
So how do I provide both Mul and Add?
My understanding is that the name is not actually used.

Reply:
Two use cases: https://github.com/pytorch/pytorch/blob/master/caffe2/python/pybind_state.cc#L1540 and https://github.com/pytorch/pytorch/blob/master/caffe2/opt/onnxifi_transformer.cc#L894 I think we should give the ONNX name here, because there is no ElementwiseLinear in ONNX. https://github.com/pytorch/pytorch/blob/master/caffe2/core/operator_schema.h#L169

Reply:
That's exactly what I mean.
All of them are using `.empty()`, i.e. only the existence matters, not content doesn't matter.

By not providing anything it's calling https://github.com/pytorch/pytorch/blob/dfb081a7e4d8cbef53084eb17968e837a825b248/caffe2/core/operator_schema.h#L169

If we want to put anything "ONNX" here, what would you suggest?
When one Caffe2 op becomes multiple ONNX op, I think usually we put the most important one here.
But in this case both `Mul` and `Add` are equally important.

Reply:
Fair enough, your argument is reasonable. Let's merge it as it is now.